helpers/lib/fonts.js

```diff
@@ -110,8 +113,9 @@ module.exports = function(format, themeSettings, handlebars) {
     // Format output based on requested format
     switch(format) {
     case 'linkElements':
+        const fontDisplay = options && options.hash['font-display'] ? options.hash['font-display'] : null;
```

Comment:
This is much better, but it's still kind of awkward that this library code needs to know about how Handlebars handles options (i.e. that some are plain variables and some are in the `hash` object. I think the `getFontsCollection` helper should translate those concerns and then pass in a normal options object to this library code. This would also act as documentation in the helper as to what options are supported (something David Chin asked for as well).

Reply:
ok I changed it again.

Reply:
Perfect